src/Components/TeamCards/TeamCards.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './teamCard.css';
+import { Card } from 'antd';
+import {
+  TwitterOutlined,
+  LinkedinOutlined,
+  GithubOutlined,
+} from '@ant-design/icons';
+
+const AboutCard = (props) => {
```

Comment:
can you change the name to TeamCard and extract the props like this
```suggestion
const TeamCard = ({ imgsrc, name, specialization }) => {
```